test/integration/install/spec.rb

```diff
@@ -0,0 +1,51 @@
+# Encoding: utf-8
+
+windows = os.windows?
+
+control 'basic_install' do
+  impact 0.4
+  title 'Install Splunk'
+  tag 'splunk_install'
+
+  describe package(os.windows? ? 'Splunk Enterprise' : 'splunk') do
```

Comment:
nit... you defined windows as `os.windows?` above.... do you want to use it here? or is it not saving you enough that you should just use os.windows? on lines 21 and 25